src/bindings.cpp

```diff
@@ -0,0 +1,68 @@
+#include <pybind11/pybind11.h>
```

Comment:
Do I understand correctly that because of the way `src/MakeApp.mk` is set up, this file is only compiled when building the python bindings, and not when nethogs is 'regularly' compiled? Perhaps it would be neat to put it in a separate directory?

Reply:
That is right, the `src/bindings.cpp` file should only be compiled when building the python bindings i.e. `pip install .`. I see `MakeApp.mk` and `MakeLib.mk` both explicitly define the list of files to be built, so keeping `src/bindings.cpp` won't affect the `make` compilation, but I see the point, I can move it.

Reply:
I could move it to a `python` directory where I can add a readme with some info (will be similar to the PR original comment). The `setup.py` and `pyproject.toml` will have to stay in the root though.

Reply:
that sounds fine to me!

I guess this makes the `contrib` script obsolete?